games-emulation/fbalpha_libretro/fbalpha_libretro-0.2.97.42.20180226git.recipe

```diff
@@ -5,7 +5,7 @@ Final Burn team and originated from FinalBurn by Dave."
 HOMEPAGE="https://github.com/libretro/fbalpha"
 COPYRIGHT="2000-2018 the Final Burn team, the libretro team"
 LICENSE="GNU GPL v3"
-REVISION="1"
+REVISION="2"
 srcGitRev="c255a8335a8eeee657ae9a0bcb528f14ccc59678"
```

Comment:
There is going to be a problem with `pkgman update` (for those users who might already have installed this package and who would not be willing to uninstall the previous version): git20180226 > 20180226git. So pkgman will think that the new version is older than the previous.

We can fix this by renaming the recipe to `fbalpha_libretro-0.2.97.42_20180226.recipe` and stick to that convention until 0.2.97.43 is out. At that point, we can come back to `fbalpha_libretro-0.2.97.43.<yyyymmdd>git.recipe`.
After all, why don't we just switch to `fbalpha_libretro-0.2.97.42_20180226.recipe` and stick to it?

While at it, shall we rename the recipe to `fbalpha_libretro-0.2.97.42_20180306.recipe` and bump to the latest snapshot?
```diff
-REVISION="2"
+REVISION="1"
-srcGitRev="c255a8335a8eeee657ae9a0bcb528f14ccc59678"
+srcGitRev="bb651e3ff70c6a4b6d93b43054a956b388e53c08"
```
(No need to bump `REVISION` because we're already "bumping" the portVersion.)

Reply:
Oops, the latest snapshot does not build yet.

Reply:
It looks like `HEAD~1` (from 2 days ago) is building fine. I've updated the comments above, in case you would like to bump to that snapshot.

Reply:
I just built bb651e3 succesfully. Does it work for you? I bumped to this revision.

Reply:
Good news: it looks like it is also building fine for me now.

At my first attempt, a couple of hours ago, I had had many errors, but I suspect that they were caused by a ramdisk filesystem corruption.
So I guess the PR is ready for a merge.

Hmm, would you like to make an extra small change?
```diff
-SOURCE_FILENAME="fbalpha-$portVersion-$srcGitRev.tar.gz"
+SOURCE_FILENAME="fbalpha-${portVersion/_/-}-$srcGitRev.tar.gz"
```
What do you think?

Reply:
Done. I am guessing I should do this for all remaining PRs as well :p